tinkerbell/smee/values.yaml

```diff
@@ -20,98 +20,104 @@ resources:
     cpu: 10m
     memory: 64Mi
 
-roleName: boots-role
-roleBindingName: boots-rolebinding
+roleName: smee-role
+roleBindingName: smee-rolebinding
 
 deployment:
   strategy:
     type: RollingUpdate
 
-service:
-  enabled: true
-  type: LoadBalancer
-  class: kube-vip.io/kube-vip-class
-  loadBalancerIP: ""
-
 # The log level for the container.
 logLevel: "info"
 
-# The network mode to launch the boots container. When true, the boots container will use the
+# The network mode to launch the smee container. When true, the smee container will use the
 # host network.
 hostNetwork: false
 
+# oneIP when defined will be used in place of the IP in the following, if they are not defined:
```

Comment:
Could we add a little description on why this is useful?

Reply:
done

Reply:
can change it or elaborate more if you would like.